managers/forms.py

```diff
@@ -5,14 +5,22 @@
 
 from bootstrap_datepicker_plus import DatePickerInput
 from django import forms
-from django.forms import SelectMultiple
+from django_select2.forms import Select2MultipleWidget
 
 from common.constants import CORRECT_DATE_FORMAT
 from managers.models import Project
 from users.models import CustomUser
 
 
-class ManagerSelectMultiple(SelectMultiple):
+class Select(Select2MultipleWidget):
+    def _get_media(self) -> forms.Media:
+        media = super()._get_media()
+        return media + forms.Media(css={"all": "employees/style.css"})
```

Comment:
Personally I would be for separating as much as possible frontend and backend layers. You could add it to the template in which this form will used. Maybe that would sound like less reusability but with current way you tightly couple this stylesheet with this form, and if someone wants to replace the frontend with different one, he still needs to mess up with backend code.

But as I have said, this is my personal approach, so I am just writing this as a suggestion.

If you still decide to go this way, you could still extract styles needed by this field/form to separate stylesheet and give it name related to the field/form.

Reply:
done!